### PR TITLE
Mm mobile 2

### PR DESCRIPTION
**Pull request by mmacedom**
Modified the Navbar/menu and based the changes on the Figma file. Aimed to be as precise as possible with the measurements. Had to use a px measurement to be precise. Let me know what you think. 

I'm thinking about pushing more commits to this PR once you push the first commit through. Let me know how many commits you'd like to have per PR so I don't over flood this.

### CODE REVIEW

**Comment by GeorgeBellTMH** [comment]
I don't see any commits from you in this one...

**Comment by mmacedom** [comment]
That's odd. I did another push and it is above. Let me know.

File: src/components/Menu/menu.scss [hunk 1]
```diff
@@ -350,7 +351,8 @@
         z-index: 1030;
         padding: 2vw;
         top: 0vh;
-        height:12vw;
+        height:19.2vw;
+        line-height: 50px;
```
Comment (GeorgeBellTMH) [inline src/components/Menu/menu.scss]: This causes some issues on small mobile when the locations item is expanded (gotta fit all the items on the screen)

File: src/components/RenderRouter/ContentItem.scss [hunk 2]
```diff
@@ -176,15 +176,19 @@
 
 @media (min-width: 320px) and (max-width: 767px) {
```
Comment (GeorgeBellTMH) [inline src/components/RenderRouter/ContentItem.scss]: I was just thinking we can probably get rid of the min-width, so it works on iPhone SE and the older phones...

Comment (mmacedom) [reply src/components/RenderRouter/ContentItem.scss]: Sounds good. It'd be a good "catch all" solution for sure. I'll remove the min-width then from the style sheets later tonight.

File: src/components/Menu/menu.scss (hunk 1 above)
Comment (mmacedom) [reply src/components/Menu/menu.scss]: Ok. I'll come up with a different solution for it.

**Comment by mmacedom** [comment]
Hey George, I just commited a number of things. They may seem like quite a few things but they are essentially changes to the mobile home page.

- Removing the mobile min-width from @media on all of the scss files under the "Render Router" folder 
- Adjusting the nav as requested with a new solution
- Adjusting the hero blackbox (H1, HR, button and the blackbox height). I strived to match the specs of Figma as much as possible. The only thing that's slightly different is the height of the box which is slightly smaller. The size on Figma seemed way too tall even when I matched padding and margins. How closely are we matching the specs? 

Let me know what you think about all of this.

**Comment by mmacedom** [comment]
@GeorgeBellTMH the last commit reflects the transfer of the styles from the HeaderItem.tsx and HeroItem.tsx to HeroItem.scss. 

After I finished the transfer, I noticed that there is no HeaderItem.scss file and also noticed that there are multiple inline className="headerItem" in the HeroItem.tsx file with varying properties. I haven't transferred those yet, however want to check with you that this class is not used anywhere else in the site apart from this file. Can you confirm? 

If it is not used anywhere else, I could add a number to the end of the "headerItem" class, i.e headerItem_1/headerItem_2 and so on, to accommodate the different properties. Thoughts on this solution?

**Pull request merged**